pyblish_lite/model.py

```diff
@@ -397,7 +397,8 @@ def update_with_result(self, result):
         self.setData(index, False, IsIdle)
         self.setData(index, False, IsProcessing)
         self.setData(index, True, HasProcessed)
-        self.setData(index, result["success"], HasSucceeded)
+        if not self.data(index, HasSucceeded):
```

Comment:
This is the only part I'm having trouble understanding. Would it be possible to elaborate on why there's an if-statement here?